src/helpers.ts

```diff
@@ -37,3 +37,22 @@ export function chunkArray<T>(array: T[], size: number) {
   }
   return result;
 }
+
+// converts timestamps from second and microsecond format to milliseconds
+export function anyTimestampToMillis(timestamp: string | number): number {
```

Comment:
nit: I feel the function is not doing what it says it's doing. Could we rename it so it does what it says it does?

```suggestion
export function convertSecondsMillisOrMicrosToMillis(timestamp: string | number): number {
```

Reply:
Sure, why not

Reply:
https://github.com/yearn/yearn-sdk/pull/255/commits/0c18a883220d25f088c3bc3e73a33e8b2384b525